src/vm/errors/vm_exception.rs

```diff
@@ -307,17 +307,17 @@ mod test {
         let program =
             program!(instruction_locations = Some(HashMap::from([(pc, instruction_location)])),);
         let runner = cairo_runner!(program);
-        let vm_excep = VmException {
+        let _vm_excep = VmException {
```

Comment:
This variable is used (l319). It shouldn't be named with an underscore prefixed name.

Reply:
get rid of this temporary value